junit_html_report/inline.js

```diff
@@ -1,14 +0,0 @@
-var fs = require('fs');
-var inline = require('web-resource-inliner');
-var inputFile = './build/index.html';
-var inputString = fs.readFileSync(inputFile, "utf8")
-
-inline.html({
-  fileContent: inputString,
-  relativeTo: "./build/"
-},
-function(err, result) {
-  if (err) { console.log(err); }
-  var kotlinResource = '../test_runner/src/main/resources/inline.html';
```

Comment:
I can't find replacement for this line. @Writhe are you sure that new implementation will update `inline.html` in `resources` automatically? I guess we don't want to do copy paste manually after update.

Reply:
Ah, good catch - it won't.
I don't think we need this file anywhere other than `resources`, so I can just tell Webpack to target that directory when building the thing. Would that be ok?

Reply:
> I can just tell Webpack to target that directory when building the thing. Would that be ok?

Yes, that will work. We don't use `inline.html` anywhere else.